Source/WebCore/dom/Document.h

```diff
@@ -696,7 +696,7 @@ class Document
     inline RefPtr<Page> protectedPage() const; // Defined in DocumentInlines.h.
     WEBCORE_EXPORT RefPtr<LocalFrame> localMainFrame() const;
     const Settings& settings() const { return m_settings.get(); }
-    Ref<Settings> protectedSettings() const;
+    Ref<const Settings> protectedSettings() const;
```

Comment:
Personally, I suspect we can get rid of `protectedSettings()` entirely and just rely on `settings()` since it is inline and returns a `const` data member. Static analysis should not require us to protect settings as a result.